provisioning/playbooks/vars/mailhog.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+mailhog_install_dir: /opt/mailhog
```

Comment:
No need to explicitly set all variables here. We can rely on defaults

The only one variable that is needed is:

```
mailhog_binary_url: https://github.com/mailhog/MailHog/releases/download/v1.0.0/MailHog_linux_amd64
```

because 0.2.1 is not the latest